docs/modules/ROOT/pages/index.adoc

```diff
@@ -90,6 +90,7 @@ The extension offers the following advantages over using the vanilla https://git
 *** Enables GraalVM native image compilation
 * Dev Services
 ** Quarkus starts embedding stores and chat memory stores for dev and test mode automatically
+** Quarkus integrates with Ollama in order to handle the lifecycle of the models users configure
```

Comment:
This sentence isn't very clear to me, should it explicitly say that it automatically pulls models?

Reply:
It also preloads the model and in the future it may be able to do more stuff (depending on what Ollama provides)

Reply:
Ok, maybe be more concrete than just "handle the lifecycle"?

Reply:
Do you have any proposals?

Reply:
PR updated